readme.txt

```diff
@@ -10,7 +10,30 @@ AI experiments and capabilities for WordPress.
 
 == Description ==
 
-AI Experiments is a plugin for testing and developing AI-powered experiments for WordPress. This plugin provides a framework for building, testing, and deploying experimental AI capabilities.
+The WordPress AI Experiments plugin brings experimental AI-powered features directly into your WordPress admin and editing experience.
+
+**What's Inside:**
+
+This plugin is built on the [AI Building Blocks for WordPress](https://make.wordpress.org/ai/2025/07/17/ai-building-blocks) initiative, combining the PHP AI Client SDK, Abilities API, and MCP Adapter into a unified experience. It serves as both a practical tool for content creators and a reference implementation for developers.
+
+**Current Features:**
+
+* **Title Generation** - Generate multiple AI-powered title suggestions for your posts with a single click. Perfect for brainstorming headlines or finding the right tone for your content.
+* **Experiment Framework** - Opt-in system that lets you enable only the AI features you want to use.
+* **Multi-Provider Support** - Works with popular AI providers like OpenAI, Google AI, and Anthropic.
+
+**Coming Soon:**
+
+We're actively developing new features to enhance your WordPress workflow:
```

Comment:
I hate stale sources of truth. Instead of predicting the future in a file we need to update, can we instead point people to some GitHub issue labels or the entire /discussions table? and keep this evergreen?